src/System.Private.CoreLib/shared/System/SpanHelpers.Byte.cs

```diff
@@ -96,6 +96,97 @@ public static int LastIndexOfAny(ref byte searchSpace, int searchSpaceLength, re
             return index;
         }
 
+        // Adapted from IndexOf(...)
+        public static unsafe bool Contains(ref byte searchSpace, byte value, int length)
+        {
+            Debug.Assert(length >= 0);
+
+            if (length == 0) return false;
```

Comment:
Added short-circuit. Is this necessary; if so should we add it to `IndexOf(…)` too?